models/x500_mono_cam_down/model.sdf

```diff
@@ -7,12 +7,21 @@
     </include>
     <include merge='true'>
       <uri>model://mono_cam</uri>
-      <pose>0 0 .10 0 1.5707 0</pose>
+      <pose>0 0 0.1 0 1.5707 0</pose>
     </include>
     <joint name="CameraJoint" type="fixed">
       <parent>base_link</parent>
       <child>mono_cam/base_link</child>
-      <pose relative_to="base_link">0 0 0 0 1.5707 0</pose>
+      <pose relative_to="base_link">0 0 0.1 0 1.5707 0</pose>
+    </joint>
+    <include merge='true'>
+      <uri>model://rangefinder</uri>
+      <pose>0.05 0 0.1 0 1.5707 0</pose>
+    </include>
+    <joint name="RangeFinderJoint" type="fixed">
+      <parent>base_link</parent>
+      <child>rangefinder/base_link</child>
+      <pose relative_to="base_link">0.05 0 0.1 0 1.5707 0</pose>
```

Comment:
I would probably add a dedicated model for optical flow.

Reply:
oh yeah totally forgot https://github.com/PX4/PX4-OpticalFlow was a thing, didn't realize this was here! thanks!

Reply:
you're saying combine range finder and camera? Or a new x500 model?

Reply:
No, new x500 model